### PR TITLE
Feature/prereg/admin

### PR DESCRIPTION
**Pull request by laurenbarker**
Features:
- click on column title to sort by column (currently only ascending) -- cannot sort by column that don't pull from the draft yet
- click on pencil to edit field, hit enter to save (currently no where to actually save those)
- click on row to be taken to the editor for that draft

### CODE REVIEW

File: website/project/views/drafts.py [hunk 1]
```diff
@@ -94,7 +94,8 @@ def get_all_draft_registrations(auth, *args, **kwargs):
             raise HTTPError(http.FORBIDDEN)
         query = query & Q('fullfills', 'in', group)
 
-    all_drafts = DraftRegistration.find(query)[:count]
+    #all_drafts = DraftRegistration.find(query)[:count]
```
Comment (laurenbarker) [inline website/project/views/drafts.py:97]: Also, the query is broken I think.

File: website/project/model.py [hunk 2]
```diff
@@ -3048,11 +3048,18 @@ class DraftRegistration(AddonModelMixin, StoredObject):
 
     fulfills = fields.StringField(list=True)
     approved = fields.BooleanField(default=False)
+    comments_sent = fields.BooleanField(default=False)
```
Comment (laurenbarker) [inline website/project/model.py:3051]: these haven't been added to the serializer

File: website/templates/admin/prereg-rows.mako [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
-    <div data-bind="text: 'false'" class="col-md-1" id="comments_sent"></div>
-    <div data-bind="text: 'false'" class="col-md-1" id="approved"></div>
-    <div data-bind="text: 'false'" class="col-md-1" id="registered"></div>
-    <div data-bind="text: 'false'" class="col-md-1" id="proof_of_pub"></div>
-    <div data-bind="text: 'false'" class="col-md-1" id="payment_sent"></div>
+    <div data-bind="text: $data.initiator.username" class="col-md-1" id="email"></div>
+    <div data-bind="text: $parent.formatTime($data.initiated)" class="col-md-1" id="begun"></div>
+    <div data-bind="text: $parent.formatTime($data.updated)" class="col-md-1" id="submitted"></div>
+    <div  class="col-md-1" id="comments_sent">
+        <span data-bind="text: $parent.commentsSent"></span><i data-bind="click: $parent.selectValue, clickBubble: false, event: {mouseover: $parent.enlargeIcon, mouseout: $parent.shrinkIcon}" style="margin-left: 10px" class="fa fa-pencil"></i>
```
Comment (leb2dg) [inline website/templates/admin/prereg-rows.mako]: the input should probably be pulled into another template and then included

File: website/templates/admin/prereg-rows.mako [hunk 4]
```diff
@@ -1,12 +1,37 @@
...
+    <div data-bind="text: $parent.formatTime($data.initiated)" class="col-md-1" id="begun"></div>
+    <div data-bind="text: $parent.formatTime($data.updated)" class="col-md-1" id="submitted"></div>
+    <div  class="col-md-1" id="comments_sent">
+        <span data-bind="text: $parent.commentsSent, attr: {class: 'comments_sent' + $index()}"></span><i data-bind="click: $parent.editItem.bind($data, 'comments_sent' + $index()), clickBubble: false, event: {mouseover: $parent.enlargeIcon, mouseout: $parent.shrinkIcon}, attr: {class: 'comments_sent' + $index() + ' fa fa-pencil'}" style="margin-left: 10px"></i>
+        <div data-bind="attr: {class: 'input_comments_sent' + $index()}, valueUpdate: 'afterkeydown', enterkey: $parent.stopEditing.bind($data, 'comments_sent' + $index())" style="display: none">
+            <div><input type="radio" data-bind="value: 'no', attr: {class: 'input_comments_sent' + $index()}, checked: $parent.commentsSent"/>No</div>
+            <div><input type="radio" data-bind="value: 'yes', attr: {class: 'input_comments_sent' + $index()}"/>Yes</div>
+        </div>
+    </div>
+    <div data-bind="text: 'no'" class="col-md-1" id="new_comments"></div>
```
Comment (laurenbarker) [inline website/templates/admin/prereg-rows.mako:14]: the ones that are set to 'no' need to get data from the draft node

File: website/templates/admin/prereg-rows.mako [hunk 5]
```diff
@@ -1,12 +1,37 @@
...
+        <div data-bind="attr: {class: 'input_proof_of_pub' + $index()}, valueUpdate: 'afterkeydown', enterkey: $parent.stopEditing.bind($data, 'proof_of_pub' + $index())" style="display: none">
+            <div><input type="radio" data-bind="value: 'no', attr: {class: 'input_proof_of_pub' + $index()}, checked: $parent.proofOfPub"/>No</div>
+            <div><input type="radio" data-bind="value: 'yes', attr: {class: 'input_proof_of_pub' + $index()}"/>Yes</div>
+        </div>
+    </div>
+    <div class="col-md-1" id="payment_sent">
+        <span data-bind="text: $parent.paymentSent, attr: {class: 'payment_sent' + $index()}"></span><i data-bind="click: $parent.editItem.bind($data, 'payment_sent' + $index()), clickBubble: false, event: {mouseover: $parent.enlargeIcon, mouseout: $parent.shrinkIcon}, attr: {class: 'payment_sent' + $index() + ' fa fa-pencil'}" style="margin-left: 10px"></i>
+        <div data-bind="attr: {class: 'input_payment_sent' + $index()}, valueUpdate: 'afterkeydown', enterkey: $parent.stopEditing.bind($data, 'payment_sent' + $index())" style="display: none">
```
Comment (laurenbarker) [inline website/templates/admin/prereg-rows.mako:26]: the input should probably be pulled out into another template

File: website/static/css/registrations.css [hunk 6]
```diff
@@ -19,6 +19,9 @@
 #fileName{
 	font-weight: 300;
 }
+.row-title{
```
Comment (laurenbarker) [inline website/static/css/registrations.css:22]: does this file even get used anymore?

**Comment by samchrisinger** [comment]
Merging this manually. Please be ready to responds to comments made here:
https://github.com/CenterForOpenScience/osf.io/pull/3476

**Pull request closed without merging**